frontend/src/routes.js

```diff
@@ -3,50 +3,55 @@ import About from "./components/About";
 import SigninRoute from "./components/OIDC/SigninRoute";
 import PostLogoutRoute from "./components/OIDC/PostLogoutRoute";
 import SilentRefreshRoute from "./components/OIDC/SilentRefreshRoute";
-import Login from "./components/Login";
 import AddMember from "./components/AddMember";
 import RegisterCompleteRoute from "./components/OIDC/RegisterCompleteRoute";
+import SelfEvaluation from "./components/SelfEvaluation";
 
 export const ROUTES = {
-  home: {
-    path: "/",
-    extraProps: { exact: true },
-    component: Home
-  },
-  login: {
-    path: "/login",
-    component: Login
-  },
-  despre: {
-    path: "/despre",
-    component: About
-  },
-  account: {
-    path: "/account",
-    component: () => "Placeholder account"
-  },
-  selfevaluation: {
-    path: "/self-evaluation",
-    component: () => "Placeholder selfevaluation"
+  base: {
+    despre: {
+      path: "/despre",
+      component: About
+    },
+    home: {
+      path: "/",
+      component: Home
+    }
+  },
+  oidc: {
+    signin: {
+      path: "/signin-oidc",
+      extraProps: { exact: true },
+      component: SigninRoute
+    },
+    postlogout: {
+      path: "/post-logout",
+      extraProps: { exact: true },
+      component: PostLogoutRoute
+    },
+    silentrefresh: {
+      path: "/silent-refresh",
+      extraProps: { exact: true },
+      component: SilentRefreshRoute
+    },
+    registercomplete: {
+      path: "/register-complete",
+      extraProps: { exact: true },
```

Comment:
```suggestion
      component: SigninRoute
    },
    postlogout: {
      path: "/post-logout",
      component: PostLogoutRoute
    },
    silentrefresh: {
      path: "/silent-refresh",
      component: SilentRefreshRoute
    },
    registercomplete: {
      path: "/register-complete",
```

As far as I know, `exact:true` is needed only when you have two routes beginning with the same path (like `/`)

Reply:
True. I have removed the extraProps